src/backend/distributed/citus--5.2-4--5.2-5.sql

```diff
@@ -0,0 +1,60 @@
+SET search_path = 'pg_catalog';
+
+/* add pg_dist_node */
+CREATE TABLE citus.pg_dist_node(
+	nodeid int NOT NULL PRIMARY KEY,
+	groupid int NOT NULL,
+	nodename text NOT NULL,
+	nodeport int NOT NULL,
+	noderack text NOT NULL,
```

Comment:
Makes sense, applied the patch. However, adding entries to pg_dist_node can cause metadata syncing problems. I would prefer if we could disable manual changes to this table (or maybe all metadata tables).